scripts/rosdistro_freeze_source

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
```

Comment:
For legacy packages matching BSD headers ala https://github.com/ros-infrastructure/rosdistro/blob/master/src/rosdistro/distribution.py#L1-L32, for new work Apache 2.0 https://www.apache.org/licenses/LICENSE-2.0

You can just put your own name(company name if it's on company time) and the year.